apps/dashboard/src/routes/(navbar-pages)/network-staking/(load-validators)/(load-staking-data)/+layout.ts

```diff
@@ -136,10 +165,32 @@ export const load: LayoutLoad = async ({ depends, parent }) => {
     })
   )
 
-  const totalXrdBalance = derived(accounts, async ($accounts) =>
-    (await Promise.all($accounts.map((a) => getXRDBalance(a.address))))
-      .reduce((acc, curr) => acc.plus(curr), new BigNumber(0))
-      .toString()
+  const totalXrdBalance = derived(
+    [xrdAddress, sharedAccountsEntityDetails],
+    async ([$xrdAddress, $details]) => {
+      if (!$xrdAddress) {
+        return '0'
```

Comment:
when is there no xrd address? should it error if that is the case?

Reply:
As far as I understand`$xrdAddress` is undefined before `network-configuration` request is finished. I don't think it should error but rather "skip" and let it refresh itself when `$xrdAddress` gets defined. I could return `'-'` or `''` but I didn't want to mess around with anything down the road which might try converting this value to number

Reply:
I think ideally we could convert `network-configuration` API call to top-level server-side call to TS-RET to get us these addresses but that kind of exceeded what was required in this PR

Reply:
ok, well at least it will in this case first return 0 and then update as soon as xrdAddress is defined.. so I think this is OK